.github/workflows/ci.yml

```diff
@@ -157,6 +172,44 @@ jobs:
         run: "poetry run invoke pylint"
     needs:
       - "build"
+  pytest_all:
```

Comment:
I don't think we need to separate out, the ci can just run with pytest-all, no? Otherwise, creating more steps in the github actions runs

Reply:
right now `pytest` only runs 1 test, and that is the check to make sure the exception is raised on the get_napalm_getters() when napalm is not installed. `pytest-all` runs every other test including testing the actual get_napalm_getters() functionality when napalm is installed. I must be missing something on what you're asking.

Reply:
something _like_
```
    strategy:
      fail-fast: true
      matrix:
        python-version: ["3.7", "3.8", "3.9", "3.10"]
        testing-type: "pytest_all"
        include:
          - python-version: "3.10"
            testing-type: "pytest"
```